modules/bitgo/test/v2/fixtures/lightning/lightning.ts

```diff
@@ -75,4 +76,40 @@ export default {
     status: 'signed',
     paymentHash: 'e93d895a78995240965a74d46225133775814b07e5203c1c440d5a6a0273d714',
   },
+
+  withdraw: {
+    transfer: {
+      entries: [
+        {
+          address: '2NCDoKg3j3TrkiRa17npwnenbpadEaw3aFS',
+          wallet: '62acd3e4d1d77600074c70829a0940d8',
+          value: -30000,
+          valueString: '-30000',
+        },
+      ],
+      id: '62f428a4a8fe2800087908e901f26ae9',
+      coin: 'tbtc',
+      wallet: '62acd3e4d1d77600074c70829a0940d8',
+      heightId: '999999999-62f428a4a8fe2800087908e901f26ae9',
+      date: '2022-08-10T21:52:36.147Z',
+      type: 'send',
+      value: -30000,
+      valueString: '-30000',
+      baseValue: -30000,
+      baseValueString: '-30000',
+      payGoFee: 0,
+      payGoFeeString: '0',
+      state: 'pendingApproval',
```

Comment:
Just a note, this fixture might have to change soon since we may switch from using `pendingApproval` for withdrawals at all. It's fine for now but just wanted point it out.